### PR TITLE
Two new hotkeys: Hitting yourself with an item & hit your inactive hand with an item

### PR DESCRIPTION
**Pull request by j-awn**
[FEATURE] [INPUT]
## About the PR 

This PR adds two new hotkeys to the default human hotkeys list: `H` lets you attack yourself with the item in your active hand, and `X` lets you attack the object in your off-hand with your active hand.

To be a bit more specific, we currently have a hotkey for `"attackself"`, which calls `src.click(W, list())`; essentially, this existing hotkey action is the same as clicking on the item in your active hand, with that item active (it ends up calling the item's `attack_self` proc). The new hotkey "selfattack", bound to "H" by default, calls `src.click(src, list())`, which ends up calling the item's `attack` proc on the user. This new hotkey does nothing if you have no item in your active hand.

The new hotkey for "usehand", bound to "X" by default, is the same as clicking on an object in your off hand. If you have nothing in your active hand, the item in your off hand will be switched to your active hand. This new hotkey does nothing if you have no item in your inactive hand.

(Edit: TG hotkey for `"usehand"` is "N" by default)

## Why's this needed? 

These hotkeys make self-surgery, bomb-making, and drinking slightly less reliant on moving the mouse, among other actions.

The `H` hotkey is **really dangerous**; you will attack yourself with the object in your active hand, even if the object hurts a lot. Maybe this shouldn't be a default hotkey? Or maybe people shouldn't run with scissors? 

## Changelog

```
(u)Jawns:
(*)New hotkeys for attacking yourself and clicking on your inactive hand with your active hand! Wow!
```

### CODE REVIEW

**Comment by TTerc** [comment]
X is the hotkey for switching hands on TG controls, and this is also the control setup I'm using

consider changing it to something else please

**Comment by Carbadox** [comment]
I like the 2nd feature but Im more skeptical of the first. I think that might be encroaching on balance, maybe? idk, i could be dumb and overthinking it. I do see it being very useful in a QoL way as well tho

**Comment by ZeWaka** [comment]
> X is the hotkey for switching hands on TG controls, and this is also the control setup I'm using
> 
> consider changing it to something else please

good thing you can rebind hotkeys and have different hotkeys for different layouts

File: code/modules/interface/keybind_style.dm [hunk 1]
```diff
@@ -188,7 +190,8 @@ var/global/list/datum/keybind_style/keybind_styles = null
 	"X" = "swaphand",
```
Comment (ZeWaka) [inline code/modules/interface/keybind_style.dm:190]: `"X" = "usehand"` will conflict with this for tg layout, you need to rebind it for tg layout

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/modules/interface/keybind_style.dm (hunk 1 above)
Comment (j-awn) [reply code/modules/interface/keybind_style.dm:190]: "N" = "usehand" is rebound below for TG

Comment (ZeWaka) [reply code/modules/interface/keybind_style.dm:190]: oh yeah the hotkey system i literally wrote myself works that way doesn't it lol

**Comment by ryebek** [comment]
Would be cool if `H` worked when nothing is equipped so you can use it to buckle yourself to a chair with the hotkey.

**Comment by Tarmunora** [comment]
re buckling: the resist/block hotkey (Z by default) will buckle you into a chair you are on top of

**Review by pali6 — APPROVED** [review]

**Pull request closed without merging**